t/unit/utils/test_json.py

```diff
@@ -70,7 +70,7 @@ def test_UUID(self):
         for constructor in constructors:
             id = constructor()
             loaded_value = loads(dumps({'u': id}))
-            assert loaded_value, {'u': id}
+            assert loaded_value == {'u': id}
```

Comment:
This was error. It should be compared, not "assert first", second argument is detailed assertion message.

Reply:
good catch